src/pages/programs/index.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from "react";
+
+import ArticleSpotlightCard from "../../components/grid-aware/ArticleSpotlightCard";
+import CommunityRepProgramBlock from "../../components/grid-aware/CommunityRepProgramBlock";
+import classroom from "../../components/grid-aware/ImageHeader/stories/classroom.png";
+import ProgramsPageLargeParagraph from "../../components/grid-aware/ProgramsPageLargeParagraph";
+import ProgramsSubTitleBlock from "../../components/grid-aware/ProgramsSubTitleBlock";
+import SFServiceGuideBlock from "../../components/grid-aware/SFServiceGuideBlock";
+import ShelterConnectBlock from "../../components/grid-aware/ShelterConnectBlock";
+
+import Layout from "../../components/layout";
+import articleSpotlightImage from "./laura-barerra-vera.png";
+export default () => (
+  <Layout>
+    <ProgramsPageLargeParagraph
+      title="Our Programs"
+      description="ShelterTech's programs are designed to suppport the underserved community experiencing homelessness and housing insecurity through WiFi connectivity and acccess to online resources."
+    />
+    <ProgramsSubTitleBlock title="ShelterTech offers the following programs:" />
+    <SFServiceGuideBlock
+      title="SF Service Guide"
+      subtitle="an online directory of human services in San Francisco"
+      description="The SF Service Guide is an online directory of human services in San Francisco that launched in November 2019 and currently serves 1,500 unique monthly users. The directory's focus is on homelessness and housing services but also covers a variety of other services, from education and legal aid to senior services and re-entry programs. Our goal is to help anyone with access to smartphone, tablet, or computer to find services they need."
+      description2="SF Service Guide is supported by a grant from SF Mayor's Office of Housing and Community Development and verified for accuracy by our data partner, the Homeless Advocacy Project of the JDC."
+      image={{
+        url: classroom,
+        alt: "Classroom of volunteer members collaborating with one another.",
+      }}
+      ctaButtons={[
+        {
+          text: "Visit Site",
+          onClick: () => console.log(true),
```

Comment:
Let's have this be an external link that takes you to https://sfserviceguide.org/.